models/example/order_calculations.sql

```diff
@@ -0,0 +1,29 @@
+with orders as (
+
+    select * from {{ ref('stg_orders') }}
+
+),
+
+orderitems as (
```

Comment:
I don't think you use the order_items CTE anywhere in this model so you can remove